src/app/pages/Home/Home.tsx

```diff
@@ -0,0 +1,184 @@
+      rating: 1.5,
+      genres: ['Action'],
+      children:
+        'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Recusandae exercitationem modi aspernatur laudantium corrupti voluptate, doloribus dolorum ut est? Deserunt, inventore natus repellat enim ab soluta necessitatibus recusandae earum molestias exercitationem nemo blanditiis accusantium ipsum odit praesentium alias veniam? Veritatis nostrum corrupti praesentium velit libero molestias dicta labore nemo quibusdam.',
+      image: 'https://de.web.img2.acsta.net/pictures/14/02/13/11/38/183661.jpg',
+      isBookmarked: false,
+    },
+    {
+      layout: 'wide',
+      title: 'Serie',
+      rating: 2.5,
+      genres: ['Action'],
+      children:
+        'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Recusandae exercitationem modi aspernatur laudantium corrupti voluptate, doloribus dolorum ut est? Deserunt, inventore natus repellat enim ab soluta necessitatibus recusandae earum molestias exercitationem nemo blanditiis accusantium ipsum odit praesentium alias veniam? Veritatis nostrum corrupti praesentium velit libero molestias dicta labore nemo quibusdam.',
+      image: 'https://images-na.ssl-images-amazon.com/images/I/91kFYg4fX3L.jpg',
+      isBookmarked: false,
+    },
+    {
+      layout: 'wide',
+      title: 'Test',
+      rating: 3.5,
+      genres: ['Action'],
+      children:
+        'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Recusandae exercitationem modi aspernatur laudantium corrupti voluptate, doloribus dolorum ut est? Deserunt, inventore natus repellat enim ab soluta necessitatibus recusandae earum molestias exercitationem nemo blanditiis accusantium ipsum odit praesentium alias veniam? Veritatis nostrum corrupti praesentium velit libero molestias dicta labore nemo quibusdam.',
+      image:
+        'https://m.media-amazon.com/images/M/MV5BMDE3MjcxNjEtODUzNi00NzdmLWFlMGMtMzhmMTg3NDdmM2IyXkEyXkFqcGdeQXVyNjYyMjE4NDY@._V1_FMjpg_UX1000_.jpg',
+      isBookmarked: false,
+    },
+  ];
+
+  const [latestCards, setLatestCards] =
+    useState<CardPropsWithoutFunction[]>(mockLatestCards);
+
+  const [topFiveCards, setTopFiveCards] =
+    useState<CardPropsWithoutFunction[]>(mockTopCards);
+
+  function handleTopFiveClick(title: string) {
+    const newTopFiveCards = topFiveCards.map((card) => {
+      if (card.title === title) {
+        card.isBookmarked = !card.isBookmarked;
+      }
+
+      return card;
+    });
+    setTopFiveCards(newTopFiveCards);
+  }
+
+  function handleLatestClick(title: string) {
+    const newLatestCards = latestCards.map((card) => {
+      if (card.title === title) {
+        card.isBookmarked = !card.isBookmarked;
+      }
+      return card;
+    });
+    setLatestCards(newLatestCards);
+  }
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.innerContainer}>
+        <div className={styles.header}>
+          <Header>Top Five</Header>
+        </div>
```

Comment:
It would be nice if we could skip this extra `div`. I think we should create an issue for that, probably a general component issue, since we'll likely run into this more often

Reply:
will be fixed in #31